### PR TITLE
Add lsds

### PR DESCRIPTION
**Pull request by pattonw**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/lsd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - cython
+    - pip
+  run:
+    - python
+    - numpy
```
Comment (dopplershift) [inline recipes/lsd/meta.yaml:26]:
```suggestion
    - {{ pin_compatible('numpy') }}
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a minor tweak needed for the numpy runtime dependency.

I also need @sheridana to confirm that they are willing to be listed as a maintainer for this recipe.

File: recipes/lsd/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{% set name = "lsds" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/funkelab/lsd/archive/v{{ version }}.tar.gz
+  sha256: b1afe5e78e1ef21fbc4b03b86de3577b74bb0533f056d75d339d9b6ce0f1c2ed
```
Comment (mkitti) [inline recipes/lsd/meta.yaml:10]:
```suggestion
{% set sha256 = "b1afe5e78e1ef21fbc4b03b86de3577b74bb0533f056d75d339d9b6ce0f1c2ed" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://github.com/funkelab/lsd/archive/v{{ version }}.tar.gz
  sha256: {{ sha256 }}
```

File: recipes/lsd/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - requests
+    - cython
+    - gunpowder
+test:
+  imports:
+    - lsd
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (mkitti) [inline recipes/lsd/meta.yaml:39]:
```suggestion
  source_files:
    - tests
  commands:
    - pip check
    - pytest tests
  requires:
    - pip
    - pytest
```

See https://conda-forge.org/docs/maintainer/adding_pkgs.html#running-unit-tests

**Review by mkitti — CHANGES_REQUESTED** [review]
You can run the unit tests here as well.

File: recipes/lsd/meta.yaml (hunk 2 above)
Comment (mkitti) [reply recipes/lsd/meta.yaml:10]: This is somewhat stylistic, but it helps concentrate version update changes at the top of the file.

**Comment by sheridana** [comment]
> Just a minor tweak needed for the numpy runtime dependency.
> 
> I also need @sheridana to confirm that they are willing to be listed as a maintainer for this recipe.

Yes this is fine. Thanks for the help!

File: recipes/lsd/meta.yaml (hunk 3 above)
Comment (pattonw) [reply recipes/lsd/meta.yaml:40]: Unfortunately the tests are not ready to be included in the CI/CD. They rely on other packages that are not yet on pypi/conda-forge

**Comment by pattonw** [comment]
Thanks for the feedback @mkitti @dopplershift. I think its ready now

File: recipes/lsd/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
+{% set name = "lsds" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "b1afe5e78e1ef21fbc4b03b86de3577b74bb0533f056d75d339d9b6ce0f1c2ed" %}
```
Comment (dopplershift) [inline recipes/lsd/meta.yaml:3]: Doing the templating for the hash actually isn't considered best practice any more. Better to just leave the hash directly in the `sha256` key.

Comment (mkitti) [reply recipes/lsd/meta.yaml:3]: My fault then. autotick bot seems to handle the situation fine, though? Is that because of the grayskull template?

Comment (dopplershift) [reply recipes/lsd/meta.yaml:3]: Yeah, the bot handles it fine, we just generally stick to templating name and version. Templating the hash doesn't have any real benefit.

Comment (mkitti) [reply recipes/lsd/meta.yaml:3]: It makes the autotick bot diffs easier to read.

Comment (dopplershift) [reply recipes/lsd/meta.yaml:3]: And makes this recipe different than 90% of the others. 🤷‍♂️

Comment (pattonw) [reply recipes/lsd/meta.yaml]: I have no strong opinions on the hash templating so I removed it for the sake of conforming with the norm. Good to go now?

**Review by dopplershift — APPROVED** [review]

**Pull request merged**